examples/fabric/v4/cloudRouterConnectivity/cloudRouter2serviceprofile/terraform.tfvars.example

```diff
@@ -0,0 +1,16 @@
+equinix_client_id      = "Equnix_Client_Id"
+equinix_client_secret  = "Equinix_Client_Secret"
+connection_name = "cloudRouter_gen_csp"
+connection_type = "IP_VC"
+notifications_type = "ALL"
+notifications_emails = ["example@equinix.com"]
+bandwidth = 50
+redundancy = "SECONDARY" #"PRIMARY" #||
```

Comment:
Good that you checked it with both, but the comment should be more descriptive that it could be either/or.

Reply:
Done